vars/iqeUtils.groovy

```diff
@@ -464,49 +464,17 @@ private def createTestStages(String appName, Map appConfig) {
         configIQE(appOptions)
     }
 
-    stage("Install red-hat-internal-envs plugin") {
-        sh "iqe plugin install red-hat-internal-envs"
-    }
-
     def pluginResults = [:]
 
-    for (plugin in appConfig["plugins"]) {
-        // Loop to install our iqe plugins before our custom_packages
-        // Check if the plugin name was given in "iqe-NAME-plugin" format or just "NAME"
-        // strip unnecessary whitespace first
-        plugin = plugin.replaceAll("\\s", "")
-
-        if (plugin ==~ /iqe-\S+-plugin.*/) plugin = plugin.replaceAll(/iqe-(\S+)-plugin/, '$1')
-
-        stage("Install iqe-${plugin}-plugin") {
-            sh "iqe plugin install ${plugin}"
-        }
-    }
-
-    if (appOptions['customPackages']) {
```

Comment:
Every installation routines must be done before running the tests during the image building.

Reply:
One of the reasons we did this before was because we needed to run a different version of platform-ui-plugin in pre-prod vs. prod. What would be the recommendation now -- to build two different image tags I guess?

Reply:
I guess the other thing is -- when differences between pre-prod vs prod pop up again -- maybe we should commit to just "feature flagging" everything in our plugins instead of needing to use different plugin versions

Reply:
>  maybe we should commit to just "feature flagging" everything in our plugins instead of needing to use different plugin versions
This looks to me like a good compromise.  We can define which feature to use within a plugin based on the `ENV_FOR_DYNACONF` env variable.